client/src/components/Button2.js

```diff
@@ -0,0 +1,25 @@
+import styled from '@emotion/styled';
+
+const Button2 = styled.button`
+  text-align: center;
+  width: 180px;
```

Comment:
It's not a good practice to set fixed height and width. Instead, the size should be generated based on the font-size and padding. Same in the input.